CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Run `history.replace` once during component mount and update to avoid URL rewriting. Refs UIIN-3099.
 * ECS | "FOLIO/MARC-shared" source is displayed in manually shared instance record. Fixes UIIN-3115.
+* The item cannot be moved to another holdings associated with another instance. Fixes UIIN-3102.
```

Comment:
Hey @OleksandrHladchenko1 
This description explains the problem statement. It should rather reflect the solution implemented in the PR. Could you please update it?